src/modules/mjml/configuration/mjml-config-container.ts

```diff
@@ -0,0 +1,63 @@
+import { MjmlConfig, SellerShopConfig } from "./mjml-config";
+
+export const getDefaultEmptyMjmlConfiguration = (): SellerShopConfig["mjmlConfiguration"] => {
```

Comment:
@lkostrowski Whats the preffered place for default configuration when using tRPC?

WDYT about the dev default settings?

Reply:
We can set it with MetadataManager during app installation?